Source/TickerCounter.swift

```diff
@@ -8,41 +8,87 @@
 
 import UIKit
 
+public enum AnimationTiming: Int {
+    case easeIn
+    case easeOut
+    case easeInEaseOut
+    case linear
+}
+
+public enum Alignment: Int {
+    case left
+    case right
+    case center
+    case fill
+}
```

Comment:
i know a lot of libs out there try to put every object in one file - i dont know why & dont agree. i'd either - 1. separate these out into their own file or 2. if this is an indication that they are part of the definition of the `TickerCounter`, then house them inside `TickerCounter`